### ISSUE TITLE
EDTA_2.2.x.yml relies on anaconda channel

### ISSUE THREAD

**Issue by SwiftSeal** [oushujun/EDTA#500]
The provided conda environment contains tagged dependencies that are specific to the anaconda channel such as `libwebp==1.3.2 h11a3e52_0`. Our HPC (and others) have disabled the anaconda and defaults channels due to anaconda's recent licence dispute: https://www.theregister.com/2024/08/08/anaconda_puts_the_squeeze_on/

I believe all unavailable depencies are available through Conda-forge etc, but their tags will need to be updated/deleted.

```
Could not solve for environment specs
The following packages are incompatible
├─ libgd ==2.3.3 h5aea950_4 is installable and it requires
│  └─ libwebp with the potential options
│     ├─ libwebp [0.5.1|0.5.2|...|1.4.0], which can be installed;
│     ├─ libwebp 1.1.0 would require
│     │  └─ libwebp-base 1.1.0 2, which does not exist (perhaps a missing channel);
│     └─ libwebp 1.3.2 would require
│        └─ libtiff >=4.6.0,<4.7.0a0 , which can be installed;
├─ libtiff ==4.5.0 h6adf6a1_2 is not installable because it conflicts with any installable versions previously reported;
├─ libwebp-base ==1.3.2 hd590300_0 is not installable because it requires
│  └─ libwebp 1.3.2 , which conflicts with any installable versions previously reported;
└─ libwebp ==1.3.2 h11a3e52_0 does not exist (perhaps a typo or a missing channel).
```

**Comment by FayeFang17** [comment]
Hey,

We tested it for some time and encounter the problem that we need libtiff 4.5.0, but only the libwebp 1.3.2 from anaconda allows libtiff 4.5.0, the ones from conda-forge require higher versions. We will keep looking for a solution, but if anyone has one already, please share and let us know! Thanks!

Best,
Faye

**Comment by oushujun** [comment]
Hey,

The freshly released version (v2.2.2, https://anaconda.org/bioconda/edta) does not rely on Anaconda anymore. Please check it out and let me know if you have any issues.

Best,
Shujun
